src/Microsoft.AspNet.Http.Features/ISession.cs

```diff
@@ -1,20 +1,20 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System;
 using System.Collections.Generic;
+using System.Threading.Tasks;
 
 namespace Microsoft.AspNet.Http.Features
 {
     public interface ISession
     {
-        void Load();
+        Task LoadAsync();
```

Comment:
Things are a little different now that it's async. How does the performant user know if they need to call it? Or do they just always call it at at the beginning of their code?